database/migrations/2024_07_31_190904_add_to_classes_table.php

```diff
@@ -0,0 +1,29 @@
+<?php
```

Comment:
this way is pretty useful in production, when it is not possible to remove the data stored in the database,,

in development, use one migration file instead is better